agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/diagnostics/DiagnosticsHelper.java

```diff
@@ -67,6 +83,18 @@ public static boolean isFunctionsRpIntegration() {
     return functionsRpIntegration;
   }
 
+  public static boolean isAksRpIntegration() {
+    return aksRpIntegration;
+  }
+
+  public static boolean isVmRpIntegration() {
+    return vmRpIntegration;
+  }
+
+  public static boolean isSpringAppRpIntegration() {
+    return springAppRpIntegration;
+  }
```

Comment:
these don't look used?

Reply:
removed. we can add it back when it's needed later.